src/main/kotlin/minesweeper/inputdata/MineGameConfig.kt

```diff
@@ -0,0 +1,19 @@
+package minesweeper.inputdata
```

Comment:
비즈니스 문제를 해결하는 클래스들은 `domain` 패키지에 위치하면 어떨까요?

```suggestion
package minesweeper.domain
```